.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: ci
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+permissions:
+  actions: read
+  contents: write
+  id-token: write
+  packages: write
+  security-events: write
+
+jobs:
+  image:
+    uses: vexxhost/github-actions/.github/workflows/image.yml@main
+    with:
+      image-ref: ghcr.io/${{ github.repository_owner }}/capo-controller-manager
```

Comment:
I'm thinking we update the possible tags here to be `0.11.x-1`

Reply:
updated the tags with 0.11.5-1